magento/magento2
# Fix issue 16315: Product save with onthefly index ignores website assignments

### Description (*)

Fix issue 16315: Product save with onthefly index ignores website assignments

### Related Pull Requests


### Fixed Issues (if relevant)

1. magento/magento2#16315: Product save with onthefly index ignores website assignments

### Manual testing scenarios (*)


1. Enable "Use Flat Catalog Product"
2. Create multiple stores
3. Enable "Update On Save" for Product Flat Data indexer
4. Remove all website assignments from a product
5. Save the product
6. Check Flat table

### Questions or comments


### Contribution checklist (*)
